ci/launch_cluster.sh

```diff
@@ -33,24 +46,36 @@ wget 'https://downloads.dcos.io/dcos-test-utils/bin/linux/dcos-launch' && chmod
 envsubst <<EOF > config.yaml
 ---
 launch_config_version: 1
-template_url: $TEMPLATE
 deployment_name: $DEPLOYMENT_NAME
-provider: aws
+platform: aws
+instance_type: m4.xlarge
```

Comment:
Why `m4.xlarge` and not `m4.large`?

Reply:
ah, true, I just copy-pasted my reference config. We can definitely work with `m4.large`